package.json

```
@@ -70,8 +70,7 @@
     "semver": "^6.3.0",
     "serve": "^11.3.0",
     "ts-jest": "^25.2.1",
-    "tsd": "^0.11.0",
-    "typescript": "^3.8.3",
+    "typescript": "^3.9.1-rc",
```

Upgrade Typescript To Latest Stable Version

```suggestion
    "typescript": "^3.9.2",
```